### ISSUE TITLE
Dialyzer error on GenConsumer

### ISSUE THREAD

**Issue by Freyskeyd** [kafkaex/kafka_ex#295]
Hello,

I tried to fix an error on Dialyzer analysis but it's weird: 

```
 The inferred return type of handle_cast/2 ({'noreply',{'bad_cast',_},_}) has nothing in common with {'noreply',_}, which is the expected return type for the callback of the 'Elixir.KafkaEx.GenConsumer' behaviour
```

I get this error on module using GenConsumer.
I tried to find a solution but I don't find any good path to follow...

i'm using: kafka_ex 0.8.3

**Comment by joshuawscott** [comment]
I ran into this as well, and am going to look into a fix today.

**Comment by joshuawscott** [comment]
@Freyskeyd this was fixed in #296 which has now been merged.
